cylc/flow/scheduler.py

```diff
@@ -1885,6 +1888,7 @@ def process_cylc_stop_point(self):
         if stoppoint is not None:
             self.options.stopcp = str(stoppoint)
             self.pool.set_stop_point(get_point(self.options.stopcp))
+            self.warnif_stopcp_gte_finalcp()
```

Comment:
Maybe call the function `validate_finalcp`, or `validate_cyclepoints`… in case we decide to have more checks in there?

Reply:
Good idea.